docs/step-attributes.md

```diff
@@ -31,3 +32,33 @@ Here are commonly seen use cases that need to be handled carefully, accounting R
 | Given Coffee costs $5.00 today | Given(@"Coffee costs \\$([\d\\.]+) today") | decimal | 5.00 |
 | Given Coffee costs $5.00 today | Given(@"Coffee costs $([\d\\.]+) today") | any type | error: will not match dollar sign |
 | Given Coffee costs $5.00 today | Given(@"Coffee costs ([\d\\.]+) today") | any type | error: will not match dollar sign |
+
+## BDD Notation
+
+Some texts will use BDD notation in the form:
+
+```
+Scenario: Adding two tuples
+  Given a1 <- tuple(3, -2, 5, 1)
+    And a2 <- tuple(-2, 3, 1, 0)
+   Then a1 + a2 = tuple(1, 1, 6, 1)
+```
+
+For the most part the *feature* files are plain text and this syntax will be acceptable. But as mentioned here, the C# attributes use regex to provide the parameterization. So the best advice is to avoid using characters considered special in regex notation. Some character conflicts will only reveal themselves fully when the Scenario is combined with the use of Examples. 
+
+Consider keeping the notation simple and avoid all special characters e.g.:
+
+```
+Scenario: Adding two tuples
+  Given a1 equals tuple 3 -2 5 1
+    And a2 equals tuple -2 3 1 0
+   Then a1 plus a2 equals tuple 1 1 6 1
+```
+
+## Feature File character encoding
```

Comment:
This section deserves its own new documentation file. Can you please create a new readme file under `/docs/` and place this section about feature file character encoding into that? Next, reference that from the main page readme file, under the See Also section, similar to other links in it.